client/src/github-info.html

```diff
@@ -0,0 +1,305 @@
+             <use xlink:href="/sprite.octicons.svg#eye"></use>
+            </svg>
+            [[data.subscribers]]
+          </div>
+          <div title="[[data.open_issues]] open issues and pull requests in this repository">
+            <svg viewBox="0 0 100 100">
+             <use xlink:href="/sprite.octicons.svg#issue-opened"></use>
+            </svg>
+            [[data.open_issues]]
+          </div>
+        </section>
+
+        <section class="collapsed dark" on-tap="_toggleExpand">
+          <h3>
+            Licensed under 
+            <a href="https://spdx.org/licenses/[[data.spdx_identifier]]" target="_blank">[[data.spdx_identifier]]</a>, 
+            [[_lastUpdated(data.activity)]]
+          </h3>
+
+          <svg id="plus-button" viewBox="0 0 100 100">
+            <use xlink:href="/sprite.octicons.svg#plus"></use>
+          </svg>
+          <svg id="minus-button" viewBox="0 0 100 100">
+            <use xlink:href="/sprite.octicons.svg#dash"></use>
+          </svg>
+        </section>
+
+        <section class="expanded dark">
+          <h3>[[data.contributors.length]] contributor[[_s(data.contributors.length)]]</h3>
+          <contributors-list contributors="[[data.contributors]]"></contributors-list>
+
+          <h3>[[_lastUpdated(data.activity)]]</h3>
+          <activity-graph activity="[[data.activity]]"></activity-graph>
+
+          <h3 hidden$="[[!data.bower.keywords.length]]">Tags</h3>
+          <div id="tags">
+            <template is="dom-repeat" items="[[data.bower.keywords]]">
+              <a href="#" tag="[[item]]" on-click="_searchTag">[[item]]</a>
+            </template>
+          </div>
+
+          <h3>Download</h3>
+          <div id="download-command" title="Download command" value="" on-tap="_selectAllInfoDownload">bower i [[data.owner]]/[[data.repo]] --save</div>
+        </section>
+
+        <a id="github-link" href="https://github.com/[[data.owner]]/[[data.repo]]" target="_blank">
+          <section class="expanded">
+            <svg viewBox="0 0 100 100"><use xlink:href="/sprite.octicons.svg#mark-github"></use></svg>
+            <div>View on GitHub</div>
+          </section>
+        </a>
+
+        <a href="https://github.com/webcomponents/beta/issues/new?title=Remove%20element%20[[owner]]/[[repo]]&body=This%20element%20should%20be%20removed%20because...&labels=User%20reports">
+          <section class="expanded">Flag element</section>
+        </a>
+      </div>
+    </template>
+  </template>
+
+  <script>
+    Polymer({
+
+      is: 'github-info',
+
+      properties: {
+        data: Object,
+      },
+
+      _dpr: function(value) {
+        return value * (window.devicePixelRatio || 1);
+      },
+
+      _s: function(n) {
+        return n == 1 ? '' : 's';
+      },
+
+      _lastUpdated: function(activity) {
+        if (!activity)
+          return;
+        var i = 0;
+        for (i = 0; i < activity.length; i++) {
+          if (activity[activity.length - 1 - i])
```

Comment:
Not that this really matters, but shouldn't this just be a single line for with this line in the exit condition?

for (i = 0; i < activity.length && !activity[activity.length - 1 - i]; i++);